CVE-2021-34868.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Expected location (2021/34xxx/CVE-2021-34868.json) of file does not match actual location (CVE-2021-34868.json)